app/src/pages/inside/common/filterEntitiesGroups/stepLevelEntities.jsx

```diff
@@ -400,7 +388,16 @@ export class StepLevelEntities extends Component {
   };
 
   getEntities = () => {
-    const { intl, filterValues, projectId, launchId, visibleFilters } = this.props;
+    const { intl, projectId, launchId, visibleFilters } = this.props;
+
+    const getTestItemAttributeValuesSearch = (project, key) => {
+      return URLS.testItemAttributeValuesSearch(projectId, launchId, key);
+    };
+
+    const getTestItemAttributeKeysSearch = () => {
```

Comment:
What the reason in you put params in the first function and avoid it in the second one?

Reply:
becouse in first function i need put second parameter - key.
in second function i don't need parameters, becouse projectId already declared in the upper scope.